snowplow-tracker/src/main/java/com/snowplowanalytics/core/emitter/Emitter.kt

```diff
@@ -303,19 +301,8 @@ class Emitter(
          */
         set(serverAnonymisation) {
             field = serverAnonymisation
-            if (!isCustomNetworkConnection && builderFinished) {
-                networkConnection = emitTimeout?.let {
-                    OkHttpNetworkConnectionBuilder(uri, context)
-                        .method(httpMethod)
-                        .tls(tlsVersions)
-                        .emitTimeout(it)
-                        .customPostPath(customPostPath)
-                        .client(client)
-                        .cookieJar(cookieJar)
-                        .serverAnonymisation(serverAnonymisation)
-                        .requestHeaders(requestHeaders)
-                        .build()
-                }
+            if (!isCustomNetworkConnection && builderFinished && networkConnection is OkHttpNetworkConnection) {
+                (networkConnection as OkHttpNetworkConnection).serverAnonymisation = true
```

Comment:
Why is this always set to true? Shouldn't we set to the value we are given (the `serverAnonymisation`)?

Reply:
😫😳🤦🏼‍♀️